### PR TITLE
Implement the logic for get next,prev,best resolution

### LINKED ISSUE #474: Implement the logic for get next,prev,best resolution
Call actions on map reducer:

-  [x] get all overlay sensor names ,get dynamic key ("SensorName" for example) from config
 - [x] actions for prev.
 - [x] actions for next.
 - [ ] actions for best.

### PR DESCRIPTION
**Pull request by yairtawil**
resolve #474

### CODE REVIEW

File: src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts [hunk 1]
```diff
@@ -1,40 +1,162 @@
...
+		spyOn(fixture.nativeElement, 'blur');
+		component.hide();
+		expect(fixture.nativeElement.blur).toHaveBeenCalled();
+	});
+
+	it('show should: save action, set {top, left} on host element(by renderer), call focus(host element) and initializeSensors', () => {
+		const elem = fixture.nativeElement;
+		spyOn(elem, 'focus');
+		spyOn(component, 'initializeSensors');
+		const action = <ContextMenuShowAction> {payload: {e: {x: 1, y: 2}}};
```
Comment (asafMasa) [inline src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts]: Check that values are different before calling 'show'
expect(elem.style.top).not().toEqual('2px');
expect(elem.style.left).not().toEqual('1px');

File: src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts [hunk 2]
```diff
@@ -1,40 +1,162 @@
...
+		expect(component.displayOverlayEvent).toHaveBeenCalledWith($event, component.filteredOverlays[1]);
 	});
 
+	it('displayOverlayEvent should get event and overlay, call stopPropagation and store dispatch', () => {
+		spyOn(store, 'dispatch');
+		const $event = <MouseEvent> jasmine.createSpyObj({stopPropagation: () => {}});
+		component.displayOverlayEvent($event, {id: '6'});
+		expect(store.dispatch).toHaveBeenCalledWith(new ContextMenuDisplayAction('6'));
+
+	});
```
Comment (asafMasa) [inline src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts]: Keep up the good tests :+1:

File: src/app/packages/map-facade/components/imageries-manager/imageries-manager.component.html [hunk 3]
```diff
@@ -15,11 +15,7 @@
 </div>
 
 
-<ansyn-context-menu
-	[(show)]="contextMenu.show"
-	[left]="contextMenu.left"
-	[top]="contextMenu.top">
-</ansyn-context-menu>
+<ansyn-context-menu #contextMenu></ansyn-context-menu>
```
Comment (asafMasa) [inline src/app/packages/map-facade/components/imageries-manager/imageries-manager.component.html]: remove view child

File: src/app/packages/overlays/effects/overlays.effects.ts [hunk 4]
```diff
@@ -17,12 +17,22 @@ import { Action, Store } from '@ngrx/store';
 import { IOverlayState } from '../reducers/overlays.reducer';
 import { ICasesState } from '../../menu-items/cases/reducers/cases.reducer';
```
Comment (asafMasa) [inline src/app/packages/overlays/effects/overlays.effects.ts]: Don't suppose to include it

File: src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts (hunk 2 above)
Comment (yairtawil) [inline src/app/packages/map-facade/components/context-menu/context-menu.component.spec.ts]: Thanks! dear @asafMasa

**Pull request merged**